target/fsl_sdmmc_common.h

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (c) 2015, Freescale Semiconductor, Inc.
```

Comment:
will these file work , if the location is `target/fsl_sdmmc_common.h` ? rather than TARGET_NXP as it's only for NXP targets?